### ISSUE TITLE
MNU while scaling image

### ISSUE THREAD

**Issue by jecisc** [guillep/pharo-core#4709]
If you open a halo on an image morph and try to scale it, you get a debugger.

Title:

Instance of ImageMorph did not understand #scaleToMatch:

Stack:

```
ImageMorph(Object)>>doesNotUnderstand: #scaleToMatch:
HaloMorph>>doScale:with:
MorphEventSubscription>>notify:from:
[ :s | result := result | ((s notify: anEvent from: sourceMorph) == true) ] in MorphicEventHandler>>notifyMorphsOfEvent:ofType:from: in Block: [ :s | result := result | ((s notify: anEvent from...etc...
Set>>do:
MorphicEventHandler>>notifyMorphsOfEvent:ofType:from:
MorphicEventHandler>>mouseMove:fromMorph:
Morph>>mouseMove:
Morph>>handleMouseMove:
```

**Comment by Ducasse** [comment]
I guess that this is the same bug I found with the rotation. 
And that thomas wrote a test for.

**Comment by Alisu** [comment]
~~For the rotation, the error raised is a DNU but it seems we have a Point with SmallFloat coordinates instead of nil.~~

~~Plus the coordinates seems to be not well calculated. ~~

Look at issue #4458

**Comment by guillep** [comment]
This seems to be fixed in Pharo 9. Can you confirm?

![image](https://user-images.githubusercontent.com/708322/78388425-455c7500-75e1-11ea-8bc7-8adebf29aa5c.png)

If so, somebody has an idea of what fixed it? I'll check...

**Comment by guillep** [comment]
Fixed by @ericbrandwein in #5446 :) Thanks a lot for the third time
